util/system/getuserinfo/userinfo_windows.go

```diff
@@ -0,0 +1,47 @@
+package getuserinfo
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"syscall"
```

Comment:
Should we use `"golang.org/x/sys/windows"` instead?

Reply:
```x/sys``` is where new stuff gets added. The ```syscall``` package only changes when those changes need to support the standard library.

The ```LookupSID()``` function from the ```syscall``` package is identical to the one in ```x/sys/windows```, and calls into a stable Windows API which, judging by Microsofts track record for backwards compatibility, will probably remain unchanged for many many years.

Unless we want to use an old version of ```go``` that does not have this function in the standard library, I think we can use ```syscall```.

If you prefer, I can change it.

Reply:
We are already using this pkg https://github.com/moby/buildkit/blob/c3eddbf95b1bb4b4577dda0f02f921567544ff8a/session/sshforward/sshprovider/agentprovider_windows.go#L13 that's why I suggested to use it instead but I don't have a strict pref.

Reply:
Then it's settled. Will replace with ```x/sys/windows``` ASAP.

Reply:
For the record, the PR and discussion I mentioned is https://github.com/microsoft/go-winio/pull/197.